Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -1729,6 +1730,14 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
     public func registeredDownloadsAssets(forBundleID bundleIdentifier: BundleIdentifier) -> [DataAsset] {
         return registeredAssets(inContexts: [DataAsset.Context.download], forBundleID: bundleIdentifier)
     }
+    
+    /// Returns a list of all the custom scripts that registered for a given `bundleIdentifier`.
+    ///
+    /// - Parameter bundleIdentifier: The identifier of the bundle to return download assets for.
+    /// - Returns: A list of all the custom scripts for the given bundle.
+    public func registeredCustomScripts(forBundleID bundleIdentifier: BundleIdentifier) -> [DataAsset] {
```

Comment:
I would prefer if this was `package` level access for now so that we don't have to make more source breaking changes to change it to not have a bundle ID parameter when https://github.com/swiftlang/swift-docc/pull/1059 lands.